packages/eslint-config-audius/package.json

```diff
@@ -21,26 +21,26 @@
     "test": "echo \"Error: run tests from root\" && exit 1"
   },
   "peerDependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.30.6",
```

Comment:
wait seriously this fixes stuff?

Reply:
nono sorry, it's this

```
    "eslint-plugin-jest": "?26.5.3",
```

we should pin versions anyway though

Reply:
the question mark is exploding things

Reply:
oh nice nice, it looks like we should probably also fix the lock-file while we're at it?

Reply:
wait what's wrong with the lock-file?

Reply:
understanding this now is b/c xcode 14 has a node 16 dep -- will wait till we just get on node 16 errwher